tests/Captioning/ConverterTest.php

```diff
@@ -16,23 +16,23 @@ public function testSubrip2WebvttConversion()
 
         $content = "WEBVTT
 
-00:00:00.000 --> 00:00:20.000
+00:00:01.000 --> 00:00:20.000
```

Comment:
I see your point, and understand the limitations, but that's also the reason why I can't accept this merge request in this state.

I can't merge code that will include regression. 
Lots of subtitles start at 00:00:00.000, I can't push this to the current release branch as it would break with those files' parsing.

This could be a lead on how to bypass the DateTime limitation: http://stackoverflow.com/questions/17089533/comparing-times-with-milliseconds-and-without-dates